src/platform/Kconfig

```diff
@@ -218,6 +218,14 @@ config CONFIG_CHERRYTRAIL_EXTRA_DW_DMA
 	  Select if you need support for all 3 DMACs versus the default 2 used
 	  in baytrail.
 
+config CANNONLAKE_LPRO
```

Comment:
It cannot be named `CANNONLAKE_LPRO` cos it sounds like cnl flavour and part of platform `choice` config.
If some config is restricted to some platforms I think it's better to just wrap it with if and name it CAVS_LPRO like:
```
if CANNONLAKE
config CAVS_LPRO ...
endif
```

Reply:
OK, renamed.